github/Repository.py

```diff
@@ -2325,15 +2325,20 @@ def get_forks(self):
     def create_fork(self, organization=github.GithubObject.NotSet):
         """
         :calls: `POST /repos/{owner}/{repo}/forks <https://docs.github.com/en/rest/reference/repos#forks>`_
-        :param organization: string or "none" or "*"
+        :param organization: the name of the organization to fork into,
```

Comment:
This should be the type, not an explanation. 
```
:param orginization: :class:`github.Organization.Organization or string
```

Reply:
I moved that into a `:type:` tag, as that was becoming quite long and an explanation as to what theb `str` case is seemed clearer.

Reply:
Yes, but it also doesn't match every other method in this library.